### PR TITLE
Docs: fix typo in transfer learning tutorial

### PR DESCRIPTION
**Pull request by AntoninBeaufort**
Fix type "evaulate" to "evaluate"

### CODE REVIEW

File: site/en/tutorials/images/transfer_learning.ipynb [hunk 1]
```diff
@@ -337,7 +337,7 @@
         "id": "s9SlcbhrarOO"
       },
       "source": [
-        "Note: These layers are active only during training, when you call `model.fit`. They are inactive when the model is used in inference mode in `model.evaulate` or `model.fit`."
+        "Note: These layers are active only during training, when you call `model.fit`. They are inactive when the model is used in inference mode in `model.evaluate` or `model.fit`."
```
Comment (8bitmp3) [inline site/en/tutorials/images/transfer_learning.ipynb:340]: Let's also make sure docs can be accessed by clicking on the methods (e.g. `Model.fit` -> https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit):

```suggestion
        "Note: These layers are active only during training, when you call `Model.fit`. They are inactive when the model is used in inference mode in `Model.evaluate` or `Model.fit`."
```

This is similar to writing `tf.keras.Model.fit`, etc

**Review by 8bitmp3 — DISMISSED** [review]
thanks @AntoninBeaufort

**Comment by 8bitmp3** [comment]
@AntoninBeaufort Can you double check your CLA https://github.com/tensorflow/docs/pull/1988/checks?check_run_id=4541595063

**Comment by AntoninBeaufort** [comment]
> @AntoninBeaufort Can you double check your CLA https://github.com/tensorflow/docs/pull/1988/checks?check_run_id=4541595063

It should be good now !

**Review by 8bitmp3 — APPROVED** [review]
A new commit made the tests run again, looks like the CLA is good.

**Pull request merged**